src/app/views/cheat-sheets/nuclear-power/nuclear-power.component.ts

```diff
@@ -132,6 +133,9 @@ export class NuclearPowerComponent implements OnInit {
         pumps = turbines * (waterPerTurbine / waterPerPump);
         steamTanks = heatExchangers / 4 * storageTanksPerReactor;
 
+        reactorEfficiency = power/(reactors*40);
```

Comment:
Magic number 40 should be `reactorPower` (from line 104: const reactorPower = this.nuclearData.reactorPower;)